### ISSUE TITLE
temperature_ is broken

### ISSUE THREAD

**Issue by oneiros-de** [munin-monitoring/contrib#746]
It uses [weather.noaa.gov](http://weather.noaa.gov/), which has been discontinued.

**Comment by Alexsae24** [comment]
yes, but all data of current weater may be fount in http://tgftp.nws.noaa.gov/weather/current/ with the same code. It will be good to have new version of that perfect plugin :-)

**Comment by oneiros-de** [comment]
The data at tgpftp is buried in html files of ca. 27k. The plugin currently parses `.txt` files. Replacing the URL is not enough.

**Comment by Alexsae24** [comment]
Just use API, I've talk about area code :-)

**Comment by eserte** [comment]
Replacing
```
my $datasource = "http://weather.noaa.gov/pub/data/observations/metar/decoded/$wcode.TXT";
```
with
```
my $datasource = "http://tgftp.nws.noaa.gov/data/observations/metar/decoded/$wcode.TXT";
```
should be enough.

(A similar change is needed for weather_temp_)

**Comment by sumpfralle** [comment]
Thank you for your comments. 
This issue was fixed in 2d95020da7ae0dccf5e3e1390bfb84b586422a1d.
